server/notification-providers/flashduty.js

```diff
@@ -59,6 +59,17 @@ class FlashDuty extends NotificationProvider {
      * @returns {string}
      */
     async postNotification(notification, title, body, monitorInfo, eventStatus) {
+        let labels = {
+            resource: this.genMonitorUrl(monitorInfo),
+            check: monitorInfo.name,
+        };
+        if (monitorInfo?.tags?.length > 0) {
```

Comment:
I think assigning the missing labels to the labels' dict would be more readable if this were using a for-loop instead of reducing with the existing labels as initial.

Reply:
@guguji5 I am unsure what you mean by the request for the re-review.

Reply:
I use the for-loop to replace the reduce to enhance readability. 

And by the way update the description. as the flashduty support uptime kuma type plugin.